src/containers/nrc-content/nrc-trend/nrc-trend-component.jsx

```diff
@@ -35,6 +35,16 @@ function Trend({
             landMarineSPI={land ? t('land SPI') : t('marine SPI')}
           />
         </p>
+        <div className={styles.areaChartLegend}>
```

Comment:
We need to add this also to the PDF:

<img width="434" alt="image" src="https://user-images.githubusercontent.com/9701591/218517316-bbc9bdd3-4381-46ea-af42-45080e9b4995.png">

Also, please remove the commented code if its not used anymore